boilerplate.html

```diff
@@ -0,0 +1,101 @@
+            function addFont() {
+                var style = document.createElement('style');
+                style.rel = 'stylesheet';
+                document.head.appendChild(style);
+                style.textContent = localStorage.firaSansWeb;
+            }
+
+            try {
+                if (localStorage.firaSansWeb) {
+                    // The font is in localStorage, we can load it directly
+                    addFont();
+                } else {
+                    // We have to first load the font file asynchronously
+                    var request = new XMLHttpRequest();
+                    request.open('GET', './src/css/fonts.css', true);
+
+                    request.onload = function() {
+                        if (request.status >= 200 && request.status < 400) {
+                            // We save the file in localStorage
+                            localStorage.firaSansWeb = request.responseText;
+
+                            // ... and load the font
+                            addFont();
+                        }
+                    };
+
+                    request.send();
+                }
+            } catch(ex) {
+                // maybe load the font synchronously for woff-capable browsers
+                // to avoid blinking on every request when localStorage is not available
+            }
+        }());
+      </script>
+    </head>
+    <body class="orange">
+      <header role="banner">
+        <section class="nav"></section>
+      </header>
+      <div class="content">
+        <div class="box">
+          <section class="cb teaser" role="textbox">
+            <aside role="complementary">
+              <br />
+              <br />
+              <br />
+              <br />
+              <nav>
+                Marzipan toffee bears cupcake gingerbread.
+              </nav>
+            </aside>
+            <article role="article">
+              <h1 role="heading">Cupcake Ipsum</h1>
+
+              <p>
+              Tart sweet powder soufflé plum tootsie dessert liquorice brownie liquorice canes dragée roll gummies.
+              </p>
+            </article>
+          </section>
+          <section class="cb" role="textbox">
+            <article role="article">
+              <h2 role="heading">Dessert Sweet Almond</h2>
+              <p>
+                Jelly-o dragée cookie beans bonbon fruitcake donut danish gummi beans cake chupa.
+              </p>
+              <p>
+                <ul role="list">
+                  <li role="listitem">
+                    gingerbread bar
+                  </li>
+                    <li role="listitem">
+                    cheesecake roll canes drops
+                  </li>
+                    <li role="listitem">
+                    cupcake beans
+                  </li>
+                </ul>
+              </p>
+            </article>
+          </section>
+        </div>
+      </div>
+      <footer role="banner">
```

Comment:
I don't think we need a role of 'banner' here, instead we should put the 'contentinfo' role here, rather than on the div below :)